### PR TITLE
Clarifies IPv6 limitations

### PR DESCRIPTION
**Pull request by matsduf**
## Purpose

If zonemaster-cli is run in a non-IPv6 setting then meaningless error messages are outputted unless IPv6 has been turned off for zonemaster-cli. Docker is usually non-IPv6.

This PR updates the document to clarify this.

## How to test this PR

This documentation only. Follow the instructions to see the behavior.

### CODE REVIEW

**Comment by matsduf** [comment]
I think it would be good to include this in the v2021.2.1 patch release. If you disagree I will move it to v2022.1.

**Review by mattias-p — APPROVED** [review]
The additions to the installation sections are great. But I feel the additions to the Invocation section are a bit excessive. The `--no-ipv6` option is already explained both above and below. From my point of view it would be better to let the Invocation section focus on getting things done.

**Review by pnax — APPROVED** [review]
> From my point of view it would be better to let the Invocation section focus on getting things done.

I have the same point of view :)

**Comment by matsduf** [comment]
@mattias-p and @pnax, after seeing the reaction of some pilot users of zonemaster-cli on Docker I think that it is needed to lift `--no-ipv6` up, and some people just jump to the invocation section and skip other parts. But I do agree that the text was too wordy, so I have reduced that somewhat. I hope that you find that to be an improvement. Please re-approve.

File: USING.md [hunk 1]
```diff
@@ -27,18 +27,29 @@ on the computer and that you can run Docker on it.
...
 
-With Docker:
+Using Docker:
+```sh
+docker run -t --rm zonemaster/cli zonemaster.net --no-ipv6
+```
+or
 ```sh
 docker run -t --rm zonemaster/cli zonemaster.net
 ```
```
Comment (pnax) [inline USING.md:55]: Since I understand Docker is not configured with IPv6 by default, I think we can leave only one command that most users will use (the one with `--no-ipv6`) and remove the second one (letting people remove the option if needed).

File: USING.md [hunk 2]
```diff
@@ -47,6 +58,10 @@ With local installation:
 ```sh
 zonemaster-cli zonemaster.net
 ```
+or
+```sh
+zonemaster-cli zonemaster.net --no-ipv6
+```
```
Comment (pnax) [inline USING.md:64]: Again, I would suggest to only keep the default (and possibly most used) command and let users tweak it to their needs.

**Review by pnax — COMMENTED** [review]
Nice, I would even suggest to go one step further and only keep the command that should work out of the box.

**Review by mattias-p — APPROVED** [review]

File: USING.md (hunk 1 above)
Comment (matsduf) [reply USING.md:55]: Can we leave as-is for now, and tweak for next release?

File: USING.md (hunk 2 above)
Comment (matsduf) [reply USING.md:64]: Can we leave as-is for now, and tweak for next release? If so, can you approve?

**Review by pnax — APPROVED** [review]
